pkg/gofr/gofr.go

```diff
@@ -94,9 +95,39 @@ func New() *App {
 
 	app.subscriptionManager = newSubscriptionManager(app.container)
 
+	// static fileserver
+	currentWd, _ := os.Getwd()
+	checkDirectory := filepath.Join(currentWd, "public")
+
+	if _, err := os.Stat(checkDirectory); !os.IsNotExist(err) {
```

Comment:
Strange logic

I think would have expected to see

```go
        if _, err := os.Stat(checkDirectory); err == nil {
```

I mean with current code, if you have any other error than not exist, you will try to serve file

Reply:
Replaced with your logic thanks